pkgs/development/tools/fermyon_spin/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, stdenv
+, fetchzip
+, autoPatchelfHook
+, zlib
+, gcc-unwrapped
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fermyon_spin";
+  version = "1.1.0";
+
+  inherit (stdenv.hostPlatform) system;
+
+  platform = {
+    x86_64-linux = "linux-amd64";
+    aarch64-linux = "linux-aarch64";
+    x86_64-darwin = "macos-amd64";
+    aarch64-darwin = "macos-aarch64";
+  }.${system} or (throw "Unsupported system: ${system}");
+
+  packageHash = {
+    x86_64-linux = "sha256-bgTj0+YwFYkPuOZNizEQG74JfXlzQVEoSSJPTvYNToQ=";
+    aarch64-linux = "sha256-ayCEoIn3RBsvhhaZG8t6Fjht1ps4uafWH+TZMANAbBo=";
+    x86_64-darwin = "sha256-e3pg5bEwruT1IQcyNvo7ZoShhPu8dqxKo/Rev4rHAcs=";
+    aarch64-darwin = "sha256-kxi40r5s3G1HmL0o4UCGquAT3MAOspO934XTxVYCpaw=";
+  }.${system} or (throw "Unsupported system: ${system}");
+
+  src = fetchzip {
+    url = "https://github.com/fermyon/spin/releases/download/v${version}/spin-v${version}-${platform}.tar.gz";
+    stripRoot = false;
+    sha256 = packageHash;
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+  ];
+
+  buildInputs = [
+    zlib
+    gcc-unwrapped
```

Comment:
You probably want stdenv.cc or so

Reply:
The package requires `libgcc_s.so.1`, otherwise the `patchelf` fails.
It's in `glibc`, `gcc-unwrapped` and a few others.
Which one would be recommended to add?

Reply:
That should be gcc-unwrapped.lib, actually

Reply:
It is updated to `gcc-unwrapped.lib`.

Why is it this particular one and not `glib`?
How do I know which one to choose for the next time?